zerver/openapi/markdown_extension.py

```diff
@@ -164,7 +165,11 @@ def render_javascript_code_example(
     else:
         config = JS_CLIENT_CONFIG.splitlines()
 
-    code_example = []
+    code_example = [
+        "{tab|js}\n",
+        "More examples and documentation can be found [here](https://github.com/zulip/zulip-js).",
+    ]
```

Comment:
I think this block may need to be in the next commit, to avoid double-including this content.

Reply:
@timabbott Indeed this breaks without the second commit. I had split it for easier review in this case.